src/integrationTest/java/uk/gov/hmcts/darts/ws/CasesApiTest.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.darts.ws;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.ImportAutoConfiguration;
+import org.springframework.boot.test.autoconfigure.webservices.server.WebServiceServerTest;
+import org.springframework.cloud.contract.wiremock.AutoConfigureWireMock;
+import org.springframework.cloud.openfeign.FeignAutoConfiguration;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.ws.test.server.MockWebServiceClient;
+import org.springframework.ws.test.server.ResponseActions;
+import org.springframework.xml.transform.StringSource;
+import uk.gov.hmcts.darts.utils.TestUtils;
+
+import java.io.IOException;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlPathEqualTo;
+import static org.springframework.ws.test.server.RequestCreators.withPayload;
+import static org.springframework.ws.test.server.ResponseMatchers.noFault;
+import static org.springframework.ws.test.server.ResponseMatchers.payload;
+
+@WebServiceServerTest
+@ImportAutoConfiguration({FeignAutoConfiguration.class})
+@ComponentScan("uk.gov.hmcts.darts")
+@ActiveProfiles("int-test")
+@SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
+@AutoConfigureWireMock(port = 8090)
+class CasesApiTest {
+    @Autowired
+    MockWebServiceClient wsClient;
+
+    @Test
+    void handlesGetCases() throws IOException {
+        String soapRequestStr = TestUtils.getContentsFromFile(
```

Comment:
but i don't want to do that. in the future, there might be different tests with different scenarios, each with their own file. is there a benefit?

Reply:
You can inject it directly into your test as a param if you prefer.  Advantage is you don't have to write your own method to load the resource file.

Reply:
i'd prefer not to. I think it would get messy if there was lots of tests, all with their files defined globally at the top of the class. my 1 test uses 3 files, and so I don't think it would take much for it to get into a mess.